server/runtime/src/main/java/org/infinispan/server/logging/Log.java

```diff
@@ -245,4 +245,7 @@
 
    @Message(value = "Invalid keystore '%s'", id = 80065)
    KeyStoreException invalidKeyStore(String filename);
+
+   @Message(value = "File-less KeyStore needs to have a defined type.", id = 80066)
```

Comment:
Instead of file-less, I think something like the following might be more descriptive for the user:

"Keystore definitions with no path must have a defined type"

Reply:
Fixed